Cargo.toml

```diff
@@ -17,12 +17,11 @@ enable_unstable_features_that_may_break_with_minor_version_bumps = []
 
 [dependencies]
 base64 = "0.21.0"
-time = { version = "0.3.3", features = ["parsing", "formatting"] }
-indexmap = "1.0.2"
+time = { version = "0.3.20", features = ["parsing", "formatting"] }
```

Comment:
I tend to not bump versions. Cargo will use the latest semver compatible version by default, and this way the version number defines the minimum supported version which I prefer.